src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -105,6 +105,9 @@ def test_use_stmts(outputdir):
       use iso_c_binding'''
     assert expected in generated_code
     assert "if (first_time) then" in generated_code
+
+    generated_code = str(psy.gen).lower()
```

Comment:
Are you deliberately doing the generation a second time to check that the same code is still generated? If so, please add a comment to say so.

Reply:
I forgot to remove these lines I was using for debugging. I deleted them now.